pkg/admission/pod_eviction_k8s_clients.go

```diff
@@ -0,0 +1,113 @@
+package admission
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/grafana/rollout-operator/pkg/util"
+	config2 "github.com/grafana/rollout-operator/pkg/zpdb"
```

Comment:
I don't think the alias is necessary:

```suggestion
	"github.com/grafana/rollout-operator/pkg/zpdb"
```